content/id/docs/templates/index.md

```diff
@@ -0,0 +1,13 @@
+---
+headless: true
+
+resources:
+- src: "*alpha*"
+  title: "alpha"
+- src: "*beta*"
+  title: "beta"
+- src: "*deprecated*"
+  title: "deprecated"
```

Comment:
actually maybe `deprecated` is better since it is in Kubernetes version convention, wdyt?

I might have to change whatever I've written that got merged to master later.

Reply:
yep, I second this